src/components/concert_card/ConcertCard.js

```diff
@@ -8,12 +9,14 @@ export const ConcertCard = ({ concert, handle }) => {
 
   return (
     <article className="concertCard" id={concert.id} concert={concert}>
-      {concert.datetime}
-      {concert.venue.name}
-      {concert.title}
-      {concert.venue.location}
+      <p className="date">
+        {dayjs(concert.datetime).format('MM/DD/YYYY   h:m')}
```

Comment:
Great job formatting date. You can also modify your data in separate data cleaning functions after the fetch call is sent and before passing them like a prop.